test/DaiJugInterestRateStrategy.t.sol

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.8.10;
+
+import "dss-test/DssTest.sol";
+
+import "../src/DaiJugInterestRateStrategy.sol";
+
+contract VatMock {
+
+    uint256 Art;
+    uint256 line;
+    uint256 public live = 1;
+
+    function ilks(bytes32) public view returns (uint256, uint256, uint256, uint256, uint256) {
+        return (Art, 10 ** 27, 0, line, 0);
+    }
+
+    function setArt(uint256 _art) external {
+        Art = _art;
+    }
+
+    function setLine(uint256 _line) external {
+        line = _line;
+    }
+
+    function cage() external {
+        live = 0;
+    }
+
+}
+
+contract JugMock {
+
+    uint256 public dsr;
+
+    function ilks(bytes32) public view returns (uint256, uint256) {
+        return (dsr, 0);
+    }
+
+    function setDSR(uint256 _dsr) external {
+        dsr = _dsr;
+    }
+
+}
+
+contract DaiMock {
+
+    uint256 public liquidity;
+
+    function balanceOf(address) external view returns (uint256) {
+        return liquidity;
+    }
+
+    function setLiquidity(uint256 _liquidity) external {
+        liquidity = _liquidity;
+    }
+
+}
+
+contract DaiJugInterestRateStrategyTest is DssTest {
+
+    VatMock vat;
+    JugMock jug;
+    DaiMock dai;
+
+    DaiJugInterestRateStrategy interestStrategy;
+    DaiJugInterestRateStrategy interestStrategyNoSubsidy;
+
+    bytes32 constant ILK = "DIRECT-SPARK-DAI";
+    bytes32 constant ILK_JUG = "ETH-C";
+    uint256 constant DSR_ONE_PERCENT = 1000000000315522921573372069;
+    uint256 constant DSR_TWO_HUNDRED_PERCENT = 1000000034836767751273470154;
+    uint256 constant BR = 11055923171930957297759999;    // DSR at 1% / 90% to get SFBR as yearly APR
+    uint256 constant RBPS = RAY / 10000;
+    uint256 constant ONE_TRILLION = 1_000_000_000_000;
+
+    function setUp() public {
+        vat = new VatMock();
+        vat.setLine(1_000_000 * RAD);
+        jug = new JugMock();
+        jug.setDSR(DSR_ONE_PERCENT);
+        dai = new DaiMock();
+
+        interestStrategy = new DaiJugInterestRateStrategy(
+            address(vat),
+            address(jug),
+            ILK_JUG,
+            ILK,
+            100 * RAY / 90,  // SFBR is defined as DSR / 90%
```

Comment:
We won't need a baseRateConversion anymore since we're using the ETH-C SFBR directly

Reply:
Removing as will cover all real values in integration testing